### DISCUSSION TITLE
Project Runs but does nothing

### DISCUSSION THREAD

**Discussion by molorane** [Pi4J/pi4j#192] [Q&A]
I implemented a simple code to turn on and off a PIN. I used two channel relay to test whether the pins does change state. However, nothing happens. But , when I run a python script without changing the connections, the pins does turn on and off. Please help. I followed this guide [https://github.com/Pi4J/pi4j-examples/blob/master/src/main/java/com/pi4j/example/gpio/digital/output/DigitalOutputExample.java](url)

The code does nothing but on the log/console I see the following
<<DIGITAL CHANGE EVENT>> [@DOUT-20 "DOUT-20" (#20)] STATE:[HIGH -> SLOW]
<<DIGITAL CHANGE EVENT>> [@DOUT-20 "DOUT-20" (#20)] STATE:[LOW-> HIHG]

which to me implies it should work.

When I run this python
`import RPi.GPIO as GPIO
import time

GPIO.setmode(GPIO.BCM)

GPIO.setup(20, GPIO.OUT)

GPIO.output(20, GPIO.LOW)
time.sleep(5)
GPIO.output(20, GPIO.HIGH)`

My code is here on Github : [https://github.com/molorane/raspberrypi-gpo-pin](url)

It's a simple spring-boot maven project running on java11. I installed java11 on the raspberrypi and i run the springboot build artifact on
commandline on the rapsberrypi

Please help, what am I doing wrong.

**Comment by FDelporte** [comment]
dear @molorane, can you please add this dependency to your pom.xml?

```
        <dependency>
            <groupId>com.pi4j</groupId>
            <artifactId>pi4j-plugin-pigpio</artifactId>
            <version>${pi4j.version}</version>
        </dependency>
```

It would also make it easier to assist if you could provide the loaded platforms and providers. A helper method to do this is documented on https://github.com/Pi4J/pi4j-example-minimal/blob/main/src/main/java/com/pi4j/example/MinimalExample.java#L101
